pandas/tslib.pyx

```diff
@@ -1098,6 +1102,10 @@ cdef class _Timestamp(datetime):
         out = get_start_end_field(np.array([self.value], dtype=np.int64), field, freqstr, month_kw)
         return out[0]
 
+    cpdef _get_date_name_field(self, field):
```

Comment:
not sure what you mean. the code can be moved directly w/o creating another function